### PR TITLE
core: force update labels on `SIGUSR2`

### PR DESCRIPTION
**Pull request by bvr-yr**
reasons behind: https://github.com/hyprwm/hyprlock/pull/150#issuecomment-1987201385

by default force update is disabled and can be enabled with:
`[update:12345:1]` or `[update:12345:true]`

are you ok with parsing like that? i think it makes more sense than config option

### CODE REVIEW

**Comment by vaxerski** [comment]
looks fine by me I think?

**Comment by bvr-yr** [comment]
wait i'll allow `update:0:true`

File: src/core/Timer.hpp [hunk 1]
```diff
@@ -19,5 +20,6 @@ class CTimer {
     std::function<void(std::shared_ptr<CTimer> self, void* data)> cb;
     void*                                                         data = nullptr;
     std::chrono::system_clock::time_point                         expires;
-    bool                                                          wasCancelled = false;
-};
\ No newline at end of file
+    bool                                                          wasCancelled     = false;
+    bool                                                          allowForceUpdate = false;
```
Comment (bvr-yr) [inline src/core/Timer.hpp:24]: or should I rename here? not to be the same as in [IWidget.hpp](https://github.com/hyprwm/hyprlock/pull/169/files#diff-6cf15676b59d5669af3c5248d657b158e129f110267c03965273f6d713b1a917R21)?

Comment (vaxerski) [reply src/core/Timer.hpp:24]: it fine

**Review by vaxerski — APPROVED** [review]
shoot to kill

**Pull request merged**